sdk/keyvault/samples/sharelink/ShareLink.csproj

```diff
@@ -4,6 +4,10 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <RootNamespace>Azure.Security.KeyVault.Storage</RootNamespace>
+    <NoWarn>
+      $(NoWarn);
```

Comment:
Hm. We don't suppress this anywhere else.

Reply:
True, and I looked into that as well. Basically, by samples being in the language repo, they pick up some globals (properties, targets, etc.) while not picking up others that match a more strict patter - a big reason I didn't want to include samples. In this case, though, I needed to link or duplicate source. Even in another repo, I imagine you'd agree that duplicating Azure.Core shared source is not ideal.

Reply:
Yes, but there is nothing that disables this warning to pick up.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/14563

Reply:
I bet it's the netcoreapp3.1 target.